stock_warehouse_calendar_orderpoint/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+
+from dateutil import relativedelta
+
+from odoo import api, fields, models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    @api.depends(
+        "rule_ids",
+        "product_id.seller_ids",
+        "product_id.seller_ids.delay",
+        "warehouse_id.orderpoint_calendar_id",
+    )
+    def _compute_lead_days(self):
+        super()._compute_lead_days()
+        # Override to use the orderpoint calendar to compute the 'lead_days_date'
+        for orderpoint in self.with_context(bypass_delay_description=True):
+            op_calendar = orderpoint.warehouse_id.orderpoint_calendar_id
+            if not op_calendar:
+                continue
+            if not orderpoint.product_id or not orderpoint.location_id:
+                orderpoint.lead_days_date = False
+                continue
+            lead_days = orderpoint._get_lead_days()
+            # Get the next planned date to execute this orderpoint
+            start_date = orderpoint._get_next_reordering_date()
```

Comment:
TODO: `_get_next_reordering_date` should return Today (like in std) instead of `False`
Goal is being able to use the `orderpoint_on_workday` feature without an orderpoint calendar.